content/workload-reduction-toolkit/address-workload-issues/curriculum-planning-and-delivery/change-lesson-observations-to-class-visits.md

```diff
@@ -0,0 +1,152 @@
+{/inset-text}
+
+## Create a culture of trust 
+
+There needs to be a culture of trust that goes both ways: 
+
+* the leadership team trust teachers to deliver high quality lessons without close, formal scrutiny 
+
+* teachers trust that SLT are there to value what they do each and every day, support them and not to make unfair judgements 
+
+* ‘a learning school’ culture becomes embedded: staff are open to feedback, want to improve their practice, know that their views are valued and want to try new things 
+
+## Change formal observations to lesson visits 
+
+Lesson observations have been replaced by informal lesson visits and this has created a permanent 'open door' culture.  
+
+### Be clear on why and when class visits happen 
+
+SLT chooses one of the teacher’s standards as a focus for the next lesson visit, aiming to cover all standards throughout the year. There isn’t usually a specific curriculum focus. 
+
+We limit the number of lesson visits to three per teacher a year. Other monitoring activities occur on a half-termly basis, to ensure leaders have a full picture of the quality of teaching provision.
+
+Notice will be given before a fortnightly lesson visit cycle starts, and it will be on the school calendar. 
+
+Teachers don’t usually know when leaders are coming in or what the focus is, so they don’t overprepare or tailor their teaching to specific criteria.  
+
+If the timing isn’t appropriate, then SLT will arrange to come back another time. 
+
+No lesson visits take place in the last week of any half term. 
+
+### How class visits work 
+
+Leaders divide up into pairs to walk around the school for about thirty minutes. They spend ten to fifteen minutes in each lesson and take minimal notes. 
+
+Feedback is given to teachers the same day. The pairs change over the year so leaders can learn from each other as well as seeing different teachers each time. Not all classes might be seen every time, but the aim is for all classes to be seen during a lesson visit fortnight. Sometimes there will be an EYFS, KS1 or KS2 focus. 
+
+All classrooms have an 'open door' policy. Teachers and children are used to leaders popping in and out regularly, so it doesn’t feel like a big deal when they come in. 
+
+### What happens in a class visit 
+
+Leaders look at books, chat to children and get a feel for the teaching. They are looking for evidence of the chosen teachers’ standard. 
+
+Leaders make a few notes if they want to, linked to the chosen teacher standard.
+
+## Give fast and informal feedback 
+
+Leaders give teachers feedback as soon as they can – this might be straight after the lesson visit or that afternoon. The idea is that the faster the feedback, the sooner the teacher can act upon it. 
+
+The feedback is short and informal and leaders are expected to: 
+* thank the teacher
+* say something they liked
+* suggest something to work on
+* suggest an area of professional development for them to look into
```

Comment:
Your bullet points need spaces around them, otherwise they'll render like this:

<img width="788" alt="Screenshot 2024-02-14 at 12 04 35" src="https://github.com/DFE-Digital/improve-school-workload-and-wellbeing/assets/18436946/4eafd6a2-23df-4ac2-8033-d41d2c5db800">



```suggestion
The feedback is short and informal and leaders are expected to: 

* thank the teacher

* say something they liked

* suggest something to work on

* suggest an area of professional development for them to look into
```